src/components/map/MapControls.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import ButtonControl from "./ButtonControl";
+import ResetViewControl from "@20tab/react-leaflet-resetview";
+import { AttributionControl } from "react-leaflet";
+
+const MapControls: React.FC = () => {
+  const attribution = `
```

Comment:
I need to find a better alternative for this. I don't want to introduce react-dom, but we may have to